Teradata/covalent
# feat(sandbox): commit

#### Test Steps

- [ ] go to `http://localhost:4200/#/components/breadcrumbs`
- Click on the 
<img width="226" alt="Screen Shot 2019-10-17 at 10 02 14 AM" src="https://user-images.githubusercontent.com/24550592/67030888-37fc0e80-f0c5-11e9-9dc2-186c8e04ba2d.png">
- It should open the respective stackblitz with the stub demo code (it's should not be working, this is just so we make sure we are on the same page for what the demo file structure should look like and provide a running example to communicate around)

#### General Tests for Every PR


##### Screenshots or link to StackBlitz/Plunker
